### PR TITLE
Improve release-notes further

### PR DESCRIPTION
**Pull request by pallavisontakke**
For docs-compliance for the upcoming release.

Disable-check: force-changelog-file
Disable-check: commit-count

### CODE REVIEW

**Review by svenklemm — APPROVED** [review]

File: .unreleased/pr_7187 [hunk 1]
```diff
@@ -1 +1 @@
-Fixes: #7187 Fix string literal length for compressed_data_info
+Fixes: #7187: Fix the string literal length for `compressed_data_info` function.
```
Comment (atovpeko) [inline .unreleased/pr_7187:1]:
```suggestion
Fixes: #7187: Fix the string literal length for the `compressed_data_info` function.
```

File: .unreleased/pr_7201 [hunk 2]
```diff
@@ -1,2 +1,2 @@
-Fixes: #7201 Use generic extension description when building apt and rpm loader packages
-Thanks: @posuch For reporting misleading extension description in the generic loader package
+Fixes: #7201: Use the generic extension description when building `apt` and `rpm` loader packages.
+Thanks: @posuch For reporting the misleading extension description in the generic loader packages.
```
Comment (atovpeko) [inline .unreleased/pr_7201:2]:
```suggestion
Thanks: @posuch for reporting the misleading extension description in the generic loader packages.
```

File: .unreleased/pr_7207 [hunk 3]
```diff
@@ -1 +1 @@
-Implements: #7207 Refactor decompress_batches_scan functions
+Implements: #7207: Refactor the `decompress_batches_scan` functions for easier meintenance.
```
Comment (atovpeko) [inline .unreleased/pr_7207:1]:
```suggestion
Implements: #7207: Refactor the `decompress_batches_scan` functions for easier maintenance.
```

File: .unreleased/pr_7227 [hunk 4]
```diff
@@ -1 +1 @@
-Fixes: #7227 Add index to compression_chunk_size catalog table
+Fixes: #7227: Add index to the `compression_chunk_size` catalog table.
```
Comment (atovpeko) [inline .unreleased/pr_7227:1]:
```suggestion
Fixes: #7227: Add an index to the `compression_chunk_size` catalog table.
```

File: .unreleased/pr_7229 [hunk 5]
```diff
@@ -1 +1 @@
-Fixes: #7229 Fix foreign key constraints where index and constraint column order differ
+Fixes: #7229: Fix the foreign key constraints where the index and constraint columns' order differ.
```
Comment (atovpeko) [inline .unreleased/pr_7229:1]:
```suggestion
Fixes: #7229: Fix the foreign key constraints where the index and the constraint column order are different.
```

**Review by atovpeko — CHANGES_REQUESTED** [review]
A few minor fixes

File: .unreleased/pr_7187 (hunk 1 above)
Comment (pallavisontakke) [reply .unreleased/pr_7187]: Alright, edited so.

File: .unreleased/pr_7201 (hunk 2 above)
Comment (pallavisontakke) [reply .unreleased/pr_7201]: Alright, edited so.

File: .unreleased/pr_7227 (hunk 4 above)
Comment (pallavisontakke) [reply .unreleased/pr_7227]: Alright, edited so.

File: .unreleased/pr_7229 (hunk 5 above)
Comment (pallavisontakke) [reply .unreleased/pr_7229]: Alright, edited so.

File: .unreleased/pr_7207 (hunk 3 above)
Comment (pallavisontakke) [reply .unreleased/pr_7207]: Alright, edited so.

**Review by atovpeko — APPROVED** [review]

**Pull request merged**